nncf/openvino/quantization/quantize.py

```diff
@@ -201,3 +206,40 @@ def quantize_with_accuracy_control_impl(model: ov.Model,
     compress_quantize_weights_transformation(quantized_model)
 
     return quantized_model
+
+
+def quantize_with_accuracy_control(model: ov.Model,
```

Comment:
What do you think about moving it to the experimental?

Reply:
Done